src/styles/ContentLayout.module.less

```diff
@@ -2,8 +2,7 @@
 
 .contentlayoutcontainer {
   width: 100vw;
-  padding: 3% 9%;
-  padding-bottom: 6%;
+  padding: @navbar-height+3rem 9%;
```

Comment:
I think you need to use `calc(..)` for this calculation to actually work. Probably why the padding looks a bit different right now.